### ISSUE TITLE
DATA Attn. Hammer Kaji Maps Updates and additions.

### ISSUE THREAD

**Issue by AskAgain** [MegaMek/megamek#2774]
Aloha Hammer, 

I was hoping you might replace my map folder within the upcoming builds with this one as I've made updates to some of the existing ones as well as added others. 

Mahalo

**Comment by HammerGS** [comment]
Did you close this for a reason?

**Comment by HammerGS** [comment]
Going to re-open pending updated maps.

**Comment by AskAgain** [comment]
@HammerGS Apologies it has taken so long for me to come back around to this. I ended up working on several other maps in the interim and will likely continue to add to them going forward.

Attached is the folder that has all my current and former maps. As I have made changes to many of the existing ones I would ask that you REPLACE the folder located in the unofficial directory with this one and not simply add it to it.

Thank you in advance sir.
[Kajis.zip](https://github.com/MegaMek/megamek/files/6408960/Kajis.zip)
